presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -339,6 +356,51 @@ public ConnectorPageSource getPageSource(
         }
     }
 
+    Optional<BitSet> getRowsFromUuid(FileSystem fileSystem, Optional<UUID> deltaShardUuid)
```

Comment:
Let's wrap this into the new page source.

Reply:
Since it's using a lot of variables in OrcStorageManager, like  ``defaultReaderAttributes``, ``orcFileTailSource``.
We don't want to burden the OrcPageSource with too many variables from OrcStorageManager.
So we will only pass it as a method reference to PageSource.